src/vs/vscode.d.ts

```diff
@@ -8688,8 +8688,9 @@ declare module 'vscode' {
 
 		/**
 		 * The debug session's name from the [debug configuration](#DebugConfiguration).
+		 * Setting the name also changes debug configuration object's `name` property.
```

Comment:
"The debug session's name is initially taken from the [debug configuration](#DebugConfiguration). Any changes will be properly reflected in the UI."

Reply:
Updated.